### PR TITLE
2 get api topics

### PR DESCRIPTION
**Pull request by liamdaly95**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const { topicData, userData, articleData, commentData } = require("../db/data/test-data/index");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("GET from non-existent endpoint", () => {
+    test("404: sends an appropriate status and error message when given a non-existent endpoint", () => {
+      return request(app)
+        .get("/api")
+        .expect(404)
```
Comment (nickkagame) [inline __tests__/app.test.js:14]: no need to test - this is express's inbuild error.  I would only include a 404 test if you've used a custom 404 handler in the app.

File: controllers/controllers.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const { selecTopics } = require("../models/models")
+
+exports.getTopics = (req, res) => {
+    selecTopics().then((topics) => {
+        res.status(200).send({topics})
+    })
```
Comment (nickkagame) [inline controllers/controllers.js:6]: pls add a catch here :) See overview comments for more details re. error handling.

**Review by nickkagame — CHANGES_REQUESTED** [review]
Nice work. Just a few bits missing. Please could you add a catch to your controller and a path to error handling middleware that'll catch a 404, and test this?
It would also be appropriate to include a 500 error handler at this stage.
Please could you resubmit? Edits should be quick. Thanks.

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the changes Liam, good to merge 🙂

**Pull request merged**